plugin/src/main/java/org/jenkinsci/plugins/casc/cli/ApplyConfigurationCommand.java

```diff
@@ -0,0 +1,34 @@
+package org.jenkinsci.plugins.casc.cli;
+
+import hudson.Extension;
+import hudson.cli.CLICommand;
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugins.casc.ConfigurationAsCode;
+import org.jenkinsci.plugins.casc.yaml.YamlSource;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.Collections;
+
+/**
+ * @author <a href="mailto:nicolas.deloof@gmail.com">Nicolas De Loof</a>
+ */
+@Extension
+public class ApplyConfigurationCommand extends CLICommand {
+
```

Comment:
WDYT about the `dryRun` flag proposal?

Reply:
A dedicated check-configuration command seems more obvious about the intent. Less risk to forget passing the option

Reply:
Concur with @ndeloof 👍 Them pesky options!